### PR TITLE
docs: update component inventory

### PR DESCRIPTION
**Pull request by hunterloftis**
## Description

`yarn build:component-inventory` (59 components now!)

### CODE REVIEW

File: INVENTORY.md [hunk 1]
```diff
@@ -106,3 +108,4 @@ and [Spectrum Web Components](https://opensource.adobe.com/spectrum-web-componen
 | typography          |                    | [📄](https://opensource.adobe.com/spectrum-css/typography.html)         | [📄](https://opensource.adobe.com/spectrum-web-components/components/styles)              |
 | underlay            |                    |                                                                         | [📄](https://opensource.adobe.com/spectrum-web-components/components/underlay)            |
 | well                | ✅                 | [📄](https://opensource.adobe.com/spectrum-css/well.html)               | ❌                                                                                        |
+| undefined           |                    |                                                                         | [📄](https://opensource.adobe.com/spectrum-web-components/components/swatch)              |
```
Comment (Westbrook) [inline INVENTORY.md:111]: This looks like the script breaks here?

Comment (Westbrook) [reply INVENTORY.md:111]: If it feels easy to address, let's do it. If not, the update is still better than the out of date version.

Comment (hunterloftis) [reply INVENTORY.md:111]: 🤦

**Review by Westbrook — APPROVED** [review]
LGTM!

**Review by Westbrook — APPROVED** [review]
Nice!

**Pull request merged**